lib/ex_doc/formatter/html.ex

```diff
@@ -44,7 +44,7 @@ defmodule ExDoc.Formatter.HTML do
   end
 
   defp normalize_config(%{main: "index"}) do
-    raise ArgumentError, message: "\"main\" cannot be set to \"index\", otherwise it will recursively link to itself"
+    raise ArgumentError, message: ~S("main" cannot be set to "index", otherwise it will recursively link to itself)
```

Comment:
I'm thiking if we should use a less common delimiter than "()" that can be easily found in message (regardless of whether this message has any parenthesis.
maybe ~S{...}  is better